### PR TITLE
scheduler_perf: allow users to specify default pod and node specs

### LINKED ISSUE #93792: new scheduler_perf framework: allow users to specify default pod and node specs
Note: this issue should be viewed in the context of https://github.com/kubernetes/kubernetes/pull/93252.

**What would you like to be added**: Each workload currently requires the user to specify the node/pod specs explicitly. While this is sometimes useful in certain workloads, for around half the workloads, all benchmarks use the same pod and node specs across all `CreateNodes` and `CreatePods` ops. It'd be nice to specify these exactly once at the head of the testCase.

**Why is this needed**: Reduce config toil.

### PR DESCRIPTION
**Pull request by sanposhiho**
Hi team!

#### What type of PR is this?

/kind feature

#### Which issue(s) this PR fixes:

Fixes #93792

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by adtac — COMMENTED** [review]
nit, lgtm otherwise

File: test/integration/scheduler_perf/scheduler_perf_test.go [hunk 1]
```diff
@@ -99,9 +99,12 @@ type testCase struct {
...
 	SchedulerConfigFile string
-	// TODO(#93792): reduce config toil by having a default pod and node spec per
-	// testCase? CreatePods and CreateNodes ops will inherit these unless
-	// manually overridden.
+	// Default path to spec file describing the pods to create. Optional.
+	// This path can be overridden in createPodsOp by setting PodTemplatePath .
+	DefaultPodTemplatePath *string
+	// Default path to spec file describing the nodes to create. Optional.
+	// This path can be overridden in createNodesOp by setting NodeTemplatePath .
+	DefaultNodeTemplatePath *string
```
[the remark this answers is not in the document]
Comment (sanposhiho) [reply test/integration/scheduler_perf/scheduler_perf_test.go]: Definitely. I deleted this feature.

**Comment by oomichi** [comment]
This seems pretty straightforward to reduce podTemplatePath places by using the default.
Please rebase this pull request on the latest master branch to solve conflicts.

/triage accepted

**Comment by sanposhiho** [comment]
@oomichi @Huang-Wei
Sorry, I have been away from here for a long time. Please take a look at this.

**Comment by Huang-Wei** [comment]
/approve
/label tide/merge-method-squash

Thanks @sanposhiho .

**Pull request merged**